examples/openthread_example/main.cpp

```diff
@@ -0,0 +1,72 @@
+#include "msg.h"
+#include <stdio.h>
+#include <openthread.h>
+#include "ot.h"
+#include "thread.h"
+#include "xtimer.h"
+#include "msg.h"
+#include "random.h"
+#include "cpuid.h"
+
+#define ENABLE_DEBUG (1)
```

Comment:
Oh, sorry. There's a lot of non PR code here that I just added for making
it work temporally.
Actually, I copied the __libc_init_array code to that file, but I have to
removed it at some point.

Cheers!

2016-06-23 17:32 GMT+02:00 Martine Lenders <notifications@github.com>:

> In examples/openthread_example/main.cpp
> <https://github.com/RIOT-OS/RIOT/pull/5552#discussion_r68257572>:
>
> > @@ -0,0 +1,72 @@
> > +#include "msg.h"
> > +#include <stdio.h>
> > +#include <openthread.h>
> > +#include "ot.h"
> > +#include "thread.h"
> > +#include "xtimer.h"
> > +#include "msg.h"
> > +#include "random.h"
> > +#include "cpuid.h"
> > +
> > +#define ENABLE_DEBUG (1)
>
> Reminder to set this to 0 when the PR reaches merge-able state ;-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/5552/files/9ba31eab2f2831ee84090c49af360d69afd1aff3#r68257572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABM8SK02tIk-7c9b22CiEhj61DhAR0ATks5qOqcUgaJpZM4I2b6I>
> .
>